packages/keyring-internal-api/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "baseUrl": "./"
+  },
+  "references": [
+    {
+      "path": "../keyring-api/tsconfig.build.json"
+    }
+  ],
+  "include": ["./src", "../keyring-api/src/api.ts"],
```

Comment:
Is it really needed to include from a different package?

Reply:
Does not seem normal no, I'll remove it too.